meta-resin-common/recipes-connectivity/resin-ntp-config/resin-ntp-config/resin-ntp-config.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Resin NTP configuration service
+DefaultDependencies=no
+Requires=mnt-boot.mount etc-systemd-timesyncd.conf.mount
+After=mnt-boot.mount etc-systemd-timesyncd.conf.mount
+Before=systemd-timesyncd.service
+
+[Service]
+ExecStart=@BASE_BINDIR@/bash @BINDIR@/resin-ntp-config
+Type=oneshot
+RemainAfterExit=yes
+StandardError=journal+console
```

Comment:
Which variable are you referring to? It's a copy and paste from resin-net-config